Makefile

```diff
@@ -313,6 +313,16 @@ package_offline: compile build modify_composefile
 		nginx:1.11.5 registry:2.5.1 photon:1.0 ; \
 	fi
 	
+	@$(TARCMD) -zcvf harbor-offline-installer-$(VERSIONTAG).tgz \
+	          --exclude=$(HARBORPKG)/common/db --exclude=$(HARBORPKG)/common/config\
+			  --exclude=$(HARBORPKG)/common/log --exclude=$(HARBORPKG)/ubuntu \

```

Comment:
We don't have directory ubuntu anymore. Besides, maybe the include is more convenient and strict？